planetary-social/nos
# Show webp images without link preview overlay; handle animation in webp images

## Issues covered
#1488 

## Description
Shows webp images without the link preview overlay, so they appear as images the same way `.jpg`, `.png`, and other images do today. Also handles animation in webp images, so they now display the "GIF" overlay when they're animated, and tapping it plays the animation. 

Perhaps we should rename "GIF" to something else or use an icon (since animated webp images are now covered by this), but it's probably not a big deal since I'd guess most people are familiar with GIF and have never heard of webp.

## How to test
1. Enable new media display in Settings
2. Navigate to a note with a static webp image: note12lhvdu7tlf92wwavleqc5snfq0t8nx8d6nyyel60apxr6gagke2spgclju
3. Observe that it looks like an image, with no dark gray overlay at the bottom.
4. Navigate to a note with an animated webp image: note17gaxqst3ppynmy0u3r3vkvm6vha6syhr8qmh3sfjkkaj32a8hy8swtq4e9
5. Observe that it has the GIF overlay 
6. Tap the GIF overlay
7. Observe that the animation begins

## Screenshots/Video

https://github.com/user-attachments/assets/2f8ea6bf-fade-4f2d-8840-a7899db189be
